src/main/java/org/spongepowered/api/world/volume/entity/EntityVolume.java

```diff
@@ -75,6 +75,14 @@
      */
     Optional<Entity> entity(UUID uuid);
 
+    /**
+     * Gets all the loaded entities in this entity volume, in no particular
+     * order.
+     *
+     * @return All entities in this volume.
+     */
+    Collection<? extends Entity> entities();
```

Comment:
Can we determine whether the collection is thread safe? Is it unmodifiable? Reusable? Any sort of information about the collection itself? (I realize that the original `entities` isn't even documented, but this is something I'd like to have some solid understanding on, whether we want the API to be returning some concrete definition of what is a returned collection)

Reply:
I'll make it unmodifiable - some could be re-used but its best to have a uniform front.